casanovo/denovo/dataloaders.py

```
@@ -240,6 +245,25 @@ def _make_dataset(
 
         return dataset
 
+    @staticmethod
+    def _get_n_spectra(dataset):
```

Ruff (0.15.4)

[warning] Missing return type annotation for staticmethod `_get_n_spectra`

(ANN205)